client/src/components/ExpandedRowDataEdit.tsx

```diff
@@ -0,0 +1,100 @@
+import React, {
+    MouseEvent,
+    useRef,
+    useState
+} from 'react';
+import {DataEditProps} from '../lib/types';
+import {fetchData} from '../services/fetch';
+import {logError} from '../services/log';
+import ExpandedRowDataView from './ExpandedRowDataView';
```

Comment:
Is this component needed here?

Reply:
Good catch.  Going to need to update the eslint to be more strict.  It pickup as a warning not error :(